Financial_Data_Report/python_code/reporting_spark_v1.py

```diff
@@ -22,12 +23,12 @@
 
 #Inject Kensu agent in spark session: Add the path to the .jar to the SparkSession
 spark = SparkSession.builder.appName("Example")\
-    .config("spark.driver.extraClassPath", "../lib/kensu-dam-spark-collector-0.17.3_spark-3.0.1.jar")\
+    .config("spark.driver.extraClassPath", "../lib/kensu-spark-collector-1.0.0-rc466_spark-3.0.1.jar")\
```

Comment:
P.S. ⚠️  storing jar inside repo is not going to scale well with time. git will have to download ALL jars ever stored here to clone the repo! so the earlier we fix it the better (or we'll need force push with history rewrite). 
I think now we can `curl` from public server!

Reply:
Very good idea

Reply:
https://github.com/kensuio-oss/kensu-public-examples/pull/11